02.calendar/calendar.js

```diff
@@ -0,0 +1,36 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+const year = parseInt(argv.y) || today.year;
+const month = parseInt(argv.m) || today.month;
+
+const first_date_of_the_month = DateTime.fromObject({
+  year: year,
+  month: month,
+});
+const first_day_of_the_month = first_date_of_the_month.weekday;
+
+/*calendarを表示*/
+console.log(
+  "      %i月 %i",
+  first_date_of_the_month.month,
+  first_date_of_the_month.year,
+);
+console.log("日 月 火 水 木 金 土");
+if (first_date_of_the_month.weekday !== 7) {
+  process.stdout.write("   ".repeat(first_day_of_the_month));
+}
+for (let i = 1; i <= first_date_of_the_month.daysInMonth; i++) {
+  const datetime = first_date_of_the_month.set({ day: i });
```

Comment:
```suggestion
for (let day = 1; day <= first_date_of_the_month.daysInMonth; day++) {
  const datetime = first_date_of_the_month.set({ day });
```

ただのループカウンタじゃなくて日にちを表していることをアピールしてもいいかも

Reply:
[🍀 refactor: 日付のループ(for文)の初期化式の変数を、日付を扱っていることがわかりやすいように変更](https://github.com/reckyy/js-practices/pull/2/commits/1194cf2145e83543b2d0845686112d053809beab)